rules/community/fleet/fleet_bad_action.py

```diff
@@ -0,0 +1,20 @@
+from stream_alert.shared.rule import rule
+# Remove disable import if no rules are disabled using the @disable decorator.
+from stream_alert.shared.rule import disable
+
+# Remove the @disable decorator to use this rule in a live deployment
+@disable
+@rule(
+    logs=['fleet:results']
+)
+def fleet_bad_action(rec):
+    """
+    author:       gavinelder
+    description:  Alert when a user carries out a bad action.
+    reference:    https://
```

Comment:
Do you have the reference link?

Reply:
Sorry can you provide more detail reference to the example fleet query ?

Let me know what exactly you would like added

Reply:
I am thinking maybe the reference about each field in the fleet logs. But I just checked the PR  (https://github.com/kolide/fleet/pull/2022/files) which supporting fleet logs sent to firehose, I don't see documentation related to log schema.
If no reference, you may just remove this line, or use `N/A`.

Reply:
Ack, Normally use reference for run-books internally.